pkg/kubelet/cm/helpers_linux.go

```diff
@@ -25,7 +25,7 @@ import (
 // GetCgroupSubsystems returns information about the mounted cgroup subsystems
 func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 	// get all cgroup mounts.
-	allCgroups, err := libcontainercgroups.GetCgroupMounts()
```

Comment:
Done. Can you think of any possible adverse affects of this that I should test for?